bin/hdfcoinc/pycbc_multiifo_page_foreground

```diff
@@ -0,0 +1,161 @@
+    format_strings = ['#', '#']
+    html_table = pycbc.results.html_table(columns, names,
+                                   format_strings=format_strings, page_size=10)
+
+    kwds = { 'title' :
+        'No more events louder than all background at this removal level.',
+        'cmd' :' '.join(sys.argv), }
+    pycbc.results.save_fig_with_metadata(str(html_table), args.output_file,
+                                         **kwds)
+    sys.exit(0)
+
+ifos = f.attrs['ifos'].split(' ')
+
+title = "Loudest Event Table"
+if h_num_rm:
+   foreground = f['foreground_h%d' % h_num_rm]
+else:
+   foreground = f['foreground']
+
+ii_sort = np.argsort(foreground['ifar'][:])[::-1]
+cols = [foreground['ifar'][:][ii_sort], foreground['ifar_exc'][:][ii_sort],
+        foreground['fap'][:][ii_sort],  foreground['fap_exc'][:][ii_sort],
+        foreground['stat'][:][ii_sort]]
+names = ['Inc. IFAR (YR)', 'Exc. IFAR (YR)', 'Inc. FAP', 'Exc. FAP ',
+         'Ranking Statistic']
+formats = ['#.###E0', '#.###E0', '#.##E0', '#.##E0', '##.###']
+
+detectors = f.attrs['ifos'].split(' ')
+
+# Get event time and number of IFOs information
+tc = [foreground[ifo + '/time'][:][ii_sort] for ifo in detectors]
+n_ifos = [np.count_nonzero(np.array(ct) > 0) for ct in zip(*tc)]
+mean_time = [np.mean(np.array(ct)[np.array(ct) > 0]) for ct in zip(*tc)]
+
+# Add GPS event time and n_ifos to the table
+cols.append(mean_time)
+names.append("End Time (GPS)")
+formats.append("#.##")
+
+cols.append(n_ifos)
+names.append("No. IFOs in coinc")
+formats.append("#")
+
+# Get template information - mass, spin etc
+tid = foreground['template_id'][:][ii_sort]
+with h5py.File(args.bank_file, 'r') as bank_f:
+    mass1 = bank_f['mass1'][:][tid]
+    mass2 = bank_f['mass2'][:][tid]
+    spin1z = bank_f['spin1z'][:][tid]
+    spin2z = bank_f['spin2z'][:][tid]
+mchirp, eta = pycbc.pnutils.mass1_mass2_to_mchirp_eta(mass1, mass2)
+cols += [mchirp, mass1, mass2, spin1z, spin2z]
+names += ['mchirp', 'm1', 'm2', 's1z', 's2z']
+formats += ['##.##', '##.##', '##.##', '##.##', '##.##']
+
+# Get time difference information
+det_two_combo= np.array(list(combinations(detectors,2)))
+tdiff = []
+tdiff_str = []
+tdiff_format =[]
+for i in range(len(det_two_combo)):
+     time_1 = np.array(foreground[det_two_combo[i,0]+'/time'][:][ii_sort])
+     time_2 = np.array(foreground[det_two_combo[i,1]+'/time'][:][ii_sort])
+     tdiff_vals = (time_1 - time_2) * 1000
+     tdiff_vals[np.logical_or(time_1 < 0, time_2 < 0)] = np.nan
+     tdiff_1 = ['%.2f' % td if not np.isnan(td) else ' ' for td in tdiff_vals]
+     tdiff.append(tdiff_1)
+     tdiff_head= '%s - %s time (ms)' % (det_two_combo[i,0], det_two_combo[i,1])
+     tdiff_str.append(tdiff_head)
+     tdiff_format.append('##.##')
+
+cols += tdiff
+names += tdiff_str
+formats += tdiff_format
+
+
+ids = {detector:foreground[detector+'/trigger_id'][:][ii_sort] for detector
+       in detectors}
+f.close()
+
+stfs = {detector: [h5py.File(stf_name, 'r') for stf_name in
```

Comment:
I will prefer a better naming of the variable
```suggestion
single_trigger_files = {detector: [h5py.File(single_trigger_file_name, 'r') for single_trigger_file_name in
```
might be long but will help others who revisit the code in future.